start.go

```diff
@@ -32,14 +33,6 @@ type Panel interface {
 	help() []string
 }
 
-// Default values for command line arguments.
-// TODO: change to os.UserHomeDir() calls
```

Comment:
globbing these made virtually no sense, only `cacheQueuePath` is called outside `getArgs()`. Plus, unglobbing them was necessary to call functions with multiple (error+some other type) return values.

Reply:
Initially I had them as global due to the absent of config file. Yeah, I think there's no reason to make them global now.